_data/ui/cron-scheduling.yml

```diff
@@ -0,0 +1,212 @@
+
+
+# -------------------------- #
+#     Expression examples    #
+# -------------------------- #
+
+examples:
+  - translation: "Run every day at midnight"
+    expression: |
+      ```
+      0 0 * * *
+      ```
+
+  - translation: "Run at midnight every day from Monday through Friday"
+    expression: |
+      ```
+      0 0 * * 1-5
+      ```
+
+  - translation: "Run once a week at 11:00PM on Sunday"
+    expression: |
+      ```
+      0 23 * * 0
+      ```
+
+  - translation: "Run every day at noon and midnight"
+    expression: |
+      ```
+      0 0,12 * * *
+      ```
+
+  - translation: "Run Tuesday through Thursday every week at 3:00AM"
+    expression: |
+      ```
+      0 3 * * 2-4
+      ```
+
+  - translation: "Run at midnight every Monday, Wednesday, and Friday"
+    expression: |
+      ```
+      0 0 * * 1,3,5
+      ```
+
+  - translation: "Run at minute 5, 6, 7, 8, 9, 10, and 30 every day in January"
+    expression: |
+      ```
+      5-10,30 * * 1 *
+      ```
+
+  - translation: "Run every first day of the month and every week on Wednesday"
+    expression: |
+      ```
+      * * 1 * 3
+      ```
+
+  - translation: "Run every 15th day of the month in June and every Thursday at 5:00PM"
+    expression: |
+       ```
+       * 17 15 6 4
+       ```
+
+# --------------------------- #
+#  Day specification examples #
+# --------------------------- #
+
+day-specification-examples:
+  - explanation: |
+      **IF**: All fields are unrestricted (contain asterisks)
+
+      **THEN**: Every day is selected
+    examples:
+      - translation: "Run every day"
+        expression: |
+          ```
+          [minute] [hour] * * *
+          ```
+  - explanation: |
+      **IF**: `day of month` or `month` are restricted and `day of week` is not
+
+      **THEN**: `day of month` or `month` specifies matching days
+    examples:
+      - translation: "Run every first day of every month"
+        expression: |
+          ```
+          [minute] [hour] 1 * *
+          ```
+
+      - translation: "Run every day of the first month (January)"
+        expression: |
+          ```
+          [minute] [hour] * 1 *
+          ```
+
+  - explanation: |
+      **IF**: Only `day of week` is restricted
+
+      **THEN**: `day of week` specifies matching days
+    examples:
+      - translation: "Run every first day of every week (Monday)"
+        expression: |
+          ```
+          [minute] [hour] * * 1
+          ```
+
+  - explanation: |
+      **IF**: `day of month` or `month` and `day of week` are restricted
+
+      **THEN**: Any day matching either `day of month`, `month` or the `day of week` are matched
+    examples:
+      - translation: "Run every first day in the first month (January) and the first day of every week (Monday)"
+        expression: |
+          ```
+          [minute] [hour] 1 1 1
+          ```
+
+      - translation: "Run every day in the first month (January) and the first day of every week (Monday)"
```

Comment:
I'm not sure if the english here properly highlights the combinatorics.

I might literally say something like "Run every day in the first month (January) and the first day of every week (Monday) including in January" but that sounds pretty awkward. Without that though I think it sounds too much like the options are exclusive.

Reply:
I can see your point here, but `every day in the first month (January)` does include the first day of every week in January.

Reply:
👍